server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java

```diff
@@ -711,6 +889,94 @@ boolean blackhole() {
                 assert blackholed || unDisconnected == false;
                 return blackholed;
             }
+
+            void heal() {
```

Comment:
I think this is a duplicate of `connect()` above (minus an assertion) although I prefer this name a bit.

Reply:
fixed in ca833e0